### PR TITLE
DE_translations

### PR DESCRIPTION
**Pull request by McHoffi**
Now it should be fine @MilosKozak

### CODE REVIEW

**Comment by MilosKozak** [comment]
👍

**Comment by AdrianLxM** [comment]
@McHoffi I wouldn't translate the languages. If you want to switch back to your own language you wouldn't find it ;)

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -619,4 +619,58 @@
...
+    <string name="invalid">Ungültig</string>
+    <string name="long_avgdelta">Langes durchschn. Delta</string>
+    <string name="mdi">ICT</string>
+    <string name="nsclientinternal">integrierter NSClient</string>
+    <string name="overview_newtempbasal_basaltype_label">Basaltyp</string>
+    <string name="pairingtimedout">Verbindung Zeit überschritten</string>
+    <string name="pairpump">Verbinde neue Pumpe</string>
+    <string name="prefs_delta_title">Delta Einstellungen</string>
+    <string name="remaininsulinalert">Restinsulin</string>
+    <string name="reuse">erneut verwenden</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-de/strings.xml]: Isn't that much too long for the button?

Comment (McHoffi) [reply app/src/main/res/values-de/strings.xml]: Oh sorry, have not thought about this 😄  Did you change the languages back to English? Doesn't AAPS make breaks, when the words are too long?

**Pull request merged**